blues/simulation.py

```diff
@@ -444,32 +496,26 @@ def simulateMD(self):
         self.nc_context.setPositions(md_state0['positions'])
         self.nc_context.setVelocities(md_state0['velocities'])
 
-    def runNCMC(self):
+    def run(self, nIter=100):
         """Function that runs the BLUES engine to iterate over the actions:
         Perform NCMC simulation, perform proposed move, accepts/rejects move,
         then performs the MD simulation from the NCMC state.
         """
+        logger.info('Running %i BLUES iterations...' % (nIter))
+        self._getSimulationInfo()
         #set inital conditions
-        def _ncmcSanityCheck():
```

Comment:
I'm reviewing fast, so not checking super carefully... Did we remove this sanity check, or just move it?

Reply:
`_ncmcSanityCheck` has been "removed", it's just a couple lines that I stuck up here: https://github.com/MobleyLab/blues/blob/5297f12a7e612f2b1e250ecc8879b606ab7c5f50/blues/simulation.py#L222